alfred-api-impl/src/test/java/eu/xenit/alfred/api/tests/metadata/AncestorsFromNonExistingNodeUnitTest.java

```diff
@@ -36,7 +36,7 @@ public void getAncestorsOfNodeTest() {
         Assertions.assertThrows(InvalidNodeRefException.class,
 ()->{
             eu.xenit.alfred.api.alfresco.metadata.NodeService alfredApiNodeService = new eu.xenit.alfred.api.alfresco.metadata.NodeService(
-                    serviceRegistry, alfredApiAlfrescoConverter);
+                    serviceRegistry, alfredApiAlfrescoConverter, null);
```

Comment:
why add an extra parameter for the constructor?

Reply:
An extra argument was added to the constructor to consistently use constructor wiring everywhere:

```
 public NodeService(ServiceRegistry serviceRegistry, AlfredApiToAlfrescoConversion alfredApiToAlfrescoConversion,
            Repository repository) {
```